tests/tasks/test_sop.py

```diff
@@ -0,0 +1,39 @@
+import unittest
+import tempfile
+import os
+from jiant.tasks.registry import REGISTRY
+
+
+class TestSOP(unittest.TestCase):
+    def setUp(self):
+        cls, _, kw = REGISTRY["wikipedia_corpus_sop"]
+        self.temp_dir = tempfile.mkdtemp()
+        self.max_seq_len = 24
+        self.SOPTask = cls(
+            os.path.join("wikipedia_corpus_sop"),
+            max_seq_len=self.max_seq_len,
+            name="wikipedia_corpus_sop",
+            tokenizer_name="roberta-large",
```

Comment:
Will this cause the model to download RoBERTa unnecessarily when running tests?

Reply:
No it won't since it only sets the self._tokenizer_name variable to be used later.